KEP: pass pod/container resources to CRI

askervin opened the pull request.

askervin committed:
KEP: pass pod/container resources to CRI

RenaudWasTaken commented:
Will try to take a stab at it tomorrow, one thing I noticed is that this doesn't talk about cpu_manager and numa_manager :)

marquiz committed:
Update KEP draft

Added existing messages (CPodSandboxConfigontainerConfig,
UpdateContainerResourcesRequest) for clarity/readability.

Changed resource arrays to maps and to have value of type Quantity in
order to match Pod spec.

Some changes/edits in the written parts.

marquiz committed:
KEP: address review comments

marquiz committed:
KEP: slightly reword the motivation section

marquiz committed:
KEP: use patch-like format for representing protobuf changes

Make the changes more obvious.

askervin committed:
Merge pull request #1 from askervin/marquiz/devel-kep

Update KEP draft
